bgpd/bgp_vty.c

```diff
@@ -3087,6 +3087,33 @@ DEFUN (no_bgp_network_import_check,
 	return CMD_SUCCESS;
 }
 
+
+DEFUN(bgp_enable_ipv6_linklocal_address_only,
```

Comment:
if you look above and below DEFUN's have a space before the opening paranthesis.  Please follow

Reply:
This is again done by git clang-format. Let me know if i need to revert it with space.